.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: ruby
```

Comment:
Same reason as [this comment](https://github.com/wfleming/vim-codeclimate/pull/5/files/3d267cb0f9474d4a29c402b883428cd8272ef652#r57476291): I was copy-pasting from somewhere else, and this worked, so I decided not to put too much time into it.